### DISCUSSION TITLE
website `$watch` example has wrong parenthesis

### DISCUSSION THREAD

**Discussion by diego-betto** [alpinejs/alpine#2427] [4. General]
Please check: the last parentheses I think should be `}` and not `)`

Regards,
Diego

**Comment by SimoTod** [comment]
You can send PRs for the docs. 
See packages/docs/src/en. 
For example, the relevant file for $watch is https://github.com/alpinejs/alpine/blob/main/packages/docs/src/en/magics/watch.md

**Reply by diego-betto** [reply]
Sorry man, my fault. The error is in the website. Any repo about it?
![2021-11-30_23-08](https://user-images.githubusercontent.com/11378282/144136139-a7d279c4-848d-441a-ac98-88593a2ccc3b.png)

**Reply by SimoTod** [reply]
Ah right, the homepage is the only page that is not generated by the docs repo. In that case, Caleb has to fix it.
